html/pfappserver/root/static.alt/src/views/Nodes/_store/index.js

```diff
@@ -244,14 +257,14 @@ const actions = {
     })
   },
   deregisterBulkNodes: ({commit}, macs) => {
-    commit('NODE_REQUEST')
```

Comment:
Why removed? Should be restored and renamed `ITEM_REQUEST`.

Reply:
Renamed to `ITEM_REQUEST` in https://github.com/inverse-inc/packetfence/pull/3273/commits/c5b70c1db1ddf46801049d3d79a699c855f47d80